src/containers/Login.js

```diff
@@ -4,14 +4,20 @@ import {useSelector, useDispatch} from 'react-redux';
 import LoginPage from '../components/LoginPage';
 import {fetchToken} from '../store/actions/login';
 import {getIsLoading, getStatusError, getToken} from '../store/selectors/login';
+import Cookies from '../helpers/cookies';
 
 function Login() {
   const dispatch = useDispatch();
   const isLoading = useSelector(getIsLoading);
   const statusError = useSelector(getStatusError);
   const token = useSelector(getToken);
 
-  if (token) {
+  if (token && !Cookies.getItem('token')) {
+    Cookies.setItem('token', token);
+    navigate('/home');
+  }
+
+  if (Cookies.getItem('token')) {
```

Comment:
You should update the store to make sure the store token and Cookies token are sync